files/default/test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+coverage
```

Comment:
I think there is still a valid point, so I'll add the flake8 to the readme as well.